pkg/arguments/arguments.go

```diff
@@ -157,14 +157,14 @@ func CheckIgnoredCCSFlags(ccs cluster.CCS) error {
 	return nil
 }
 
-func AddBYOVPCFlags(fs *pflag.FlagSet, value *cluster.BYOVPC) {
+func AddExistingVPCFlags(fs *pflag.FlagSet, value *cluster.ExistingVPC) {
 	fs.BoolVar(
 		&value.Enabled,
-		"byo-vpc",
+		"use-existing-vpc",
```

Comment:
We use "Install into an existing VPC" in ROSA if we want to keep it consistent across all the cli we manage